Textual fixes for the README. (#75)
matomo-org/matomo-nginx: README.md

This is a small nginx configuration that should help you get your own Matomo instance running and start collecting your own analytics.

## I already know nginx

In this case it should be enough to just take `sites-available/matomo.conf`, check if everything is configured as you like it and enable the config.

## I want to get started

- Clone this repository or [download it as a zip](https://github.com/matomo-org/matomo-nginx/archive/master.zip) then move its contents to `/etc/nginx/` (or wherever you store your nginx config).
- Read through `sites-available/matomo.conf` and modify it to fit your use case:
	- Set `server_name` to the domain(s) of your Matomo instance

> 	- Set the path to your SSL certificate (I recommend you make sure your Matomo instance is reachable only via HTTPS. If you don't have an SSL certificate for your domain yet, check out [Let's Encrypt](https://letsencrypt.org/).)

Reviewer: ```suggestion
	- Set the path to your SSL certificate (We recommend you make sure your Matomo instance is reachable only via HTTPS. If you don't have an SSL certificate for your domain yet, check out [Let's Encrypt](https://letsencrypt.org/).)
```

> 	- Do you want to support old browsers? Then you'll need to modify `ssl.conf` according to your need. (The [Mozilla SSL Config Generator](https://mozilla.github.io/server-side-tls/ssl-config-generator/) can help you.)

Reviewer: ```suggestion
	- In case you want to support old browsers, you'll need to modify `ssl.conf` according to your needs. The [Mozilla SSL Config Generator](https://mozilla.github.io/server-side-tls/ssl-config-generator/) can help you.
```
